### PR TITLE
Adds base relayer

### PR DESCRIPTION
**Pull request by albertein**
The Relayer class is not expected to be used by clients, instead the clients
will use wrappers according their stack. The actual wrappers will come
soon.

- [x] @pablasso 
- [ ] @rpfernando 

Note, the build will fail since won't add test yet, but the linter should succeed.

### CODE REVIEW

File: relayer/__init__.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from kafka import KafkaProducer
+
+from .event_emitter import EventEmitter
+from .exceptions import NonJSONSerializableMessageError, UnsupportedPartitionKeyTypeError, ConfigurationError  # noqa
```
Comment (pablasso) [inline relayer/__init__.py]: Why are we not using all of them?

Comment (albertein) [inline relayer/__init__.py]: We're using them but not on this file, those are being exported so they can easily being imported by the client.

Comment (rpfernando) [inline relayer/__init__.py]: but the client should use `from relayer.exceptions import exceptions, i_use`?

Comment (albertein) [inline relayer/__init__.py]: Cool, no problem, will change then.

File: relayer/utils/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+def get_elapsed_time_in_milliseconds(start_time, end_time):
+    elapsed_time = end_time - start_time
+    return elapsed_time.microseconds / 1000.0 + elapsed_time.seconds * 1000
```
Comment (rpfernando) [inline relayer/utils/__init__.py]: I really don't get this formula, I feel there is double time.
What does `some.microseconds` and `some.seconds` return?

Comment (albertein) [inline relayer/utils/__init__.py]: Read https://docs.python.org/3/library/datetime.html#timedelta-objects

The values are normalised such as:

```
0 <= microseconds < 1000000
0 <= seconds < 3600*24 (the number of seconds in one day)
-999999999 <= days <= 999999999
```

If that isn't clear enough, you can check it on the console:
```>>> from datetime import datetime
>>> t1 = datetime.utcnow()
>>> t2 = datetime.utcnow()
>>> t2 - t1
datetime.timedelta(0, 8, 529430)
>>> td = t2 - t1
>>> td
datetime.timedelta(0, 8, 529430)
>>> td.seconds
8
>>> td.microseconds
529430```

Look on there are `8` seconds on the timedelta but also `529430` microseconds, the microseconds are normalized so you only see them as fractions of a second, if the microseconds accounted for the total of microseconds the number would be something like `8529430` which means, 8 seconds and a half

Comment (albertein) [inline relayer/utils/__init__.py]: So te code divides microseconds by a thousand to convert them to milliseconds, and multiplies the seconds by 1000 to get milliseconds again, those are added together to get the total number of milliseconds.

**Comment by albertein** [comment]
@pablasso @rpfernando Added tests, one of them tests the `get_elapsed_time_in_milliseconds` which proves the method is correct.

**Comment by pablasso** [comment]
:shipit:

**Pull request merged**